client/my-sites/checkout/checkout-thank-you/transfer-pending/index.tsx

```diff
@@ -0,0 +1,130 @@
+/**
+ * External dependencies
+ */
+import page from 'page';
+import * as React from 'react';
+import { sprintf } from '@wordpress/i18n';
+import { useI18n } from '@wordpress/react-i18n';
+import { useDispatch, useSelector } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { getSiteSlug } from 'calypso/state/sites/selectors';
+import getAtomicTransfer from 'calypso/state/selectors/get-atomic-transfer';
+import { errorNotice } from 'calypso/state/notices/actions';
+import { transferStates } from 'calypso/state/atomic-transfer/constants';
+import { hideMasterbar, showMasterbar } from 'calypso/state/ui/masterbar-visibility/actions';
+import { useInterval } from '../../../../lib/interval/use-interval';
```

Comment:
Minor, but the path here could be `calypso/lib/interval` which might be clearer? 🤷  (The `use-interval` part is not needed because the directory re-exports it like a package might.)

Reply:
TIL, thanks! Updated here: https://github.com/Automattic/wp-calypso/pull/55073/commits/8b6fd0e0f6abb6c9f8277140e82e5b8061ca234d